docs/markdown/Using Pants/concepts/targets.md

```diff
@@ -380,11 +380,27 @@ If multiple fields are parametrized, a target will be created for each value in
 python_test(
     name="tests",
     source="tests.py",
-    interpreter_constraints=parametrize(py2=["==2.7.*"], py3=[">=3.6"]),
+    interpreter_constraints=parametrize(py2=["==2.7.*"], py3=[">=3.6,<3.7"]),
     resolve=parametrize("lock-a", "lock-b"),
 )
 ```
 
+To parametrize multiple fields together as one parametrization, splat a named parametrize object to the target with the field values to use for that group as the parametrization keyword arguments. (See example below.)  This is useful to avoid a full cartesian product if not every combination of field values makes sense. i.e. The previous example uses the same resolve (lockfile) for both interpreter constraints, however if you want to use a different resolve per interpreter, then grouping the resolve value with the interpreter constraint may be the way to go.
```

Comment:
The Python docs refer to this as arg or kwarg "unpacking" not "splatting".
https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists
https://stackoverflow.com/a/2322384

Reply:
hehe.. I'll re-phrase